src/check_allele_table.py

```diff
@@ -35,8 +35,16 @@
         assert all(b in 'ACGTN' for b in record['ref']), 'Unexpected "ref" column value: ' + record['ref']
         assert all(b in 'ACGTN' for b in record['alt']), 'Unexpected "alt" column value: ' + record['alt']  # there's one clinvar record with ALT = "NTGT". Not sure how to handle it.
         assert record['measureset_type'] in [
-            "Variant", "Haplotype", "CompoundHeterozygote", "Phase unknown", "Distinct chromosomes",
-            "CompoundHeterozygote;Haplotype", "Variant;gene-variant", "Haplotype;Phase unknown", "Haplotype;Variant"
+            "Variant",
+            "Haplotype",
+            "CompoundHeterozygote",
+            "Phase unknown",
+            "Distinct chromosomes",
+            "CompoundHeterozygote;Haplotype",
+            "Variant;gene-variant",
+            "Haplotype;Phase unknown",
+            "Haplotype;Variant",
+            "Haplotype, single variant",
```

Comment:
It looks like there's inconsistency in the punctuation and spacing of these incoming types.
Are these coming from the VEP data and therefore outside of our control ?

Reply:
Yes, they are. We don't have control on these fields.

Reply:
OK happy to approve